ivy_tests/test_ivy/test_frontends/test_mindspore/test_ops/test_function/test_mindspore_nn_func.py

```diff
@@ -77,3 +77,46 @@
 #         on_device=on_device,
 #         x=x[0],
 #     )
+
+# gumbel_softmax
+# @handle_frontend_test(
+#     fn_tree="mindspore.ops.function.nn_func.gumbel_softmax",
+#     logits=helpers.dtype_and_values(
+#         available_dtypes=helpers.get_dtypes("valid"),
+#         num_arrays=1,
+#         shared_dtype=True,
+#     ),
+#     tau=st.floats(min_value=0.1, max_value=10.0),
+#     hard=st.booleans(),
+#     eps=st.floats(min_value=1e-12, max_value=1e-6),
+#     dim=st.integers(min_value=-1, max_value=1),
+# )
+# def test_mindspore_gumbel_softmax(
+#     *,
+#     logits,
+#     tau,
+#     hard,
+#     eps,
+#     dim,
+#     on_device,
+#     fn_tree,
+#     frontend,
+#     test_flags,
+# ):
+#     input_logits = logits
+#     gumbel_tau = tau
+#     gumbel_hard = hard
+#     gumbel_eps = eps
+#     gumbel_dim = dim
+
+#     helpers.test_frontend_function(
```

Comment:
we also need to include `test_values=False` here, ala https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_non_linear_activation_functions.py#L1398. also need to include `input_dtypes`. see the other tests in this file for reference, and make sure you're including everything required.